src/Sulu/Component/Webspace/Loader/Exception/InvalidErrorTemplateException.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Loader\Exception;
+
+/**
+ * Represents a wrong error template configuration
+ */
+class InvalidErrorTemplateException extends WebspaceException
+{
+    /**
+     * @var string
+     */
+    private $template;
+
+    /**
+     * InvalidErrorTemplateException constructor.
+     * @param string $template
+     * @param string $webspace
+     */
+    public function __construct($template, $webspace)
+    {
+        parent::__construct(
+            sprintf(
+                'Error template "%s" definition in "%s". It has to be defined as default or with a code.',
```

Comment:
`Error template "%s" in webspace "%s" has to be defined as default or with a code.`